### DISCUSSION TITLE
Plex watchlist and jellyseerr not working

### DISCUSSION THREAD

**Discussion by faulkkev** [seerr-team/seerr#1003] [Q&A]
I am trying to enable the feature. I have plex/sonarr/radarr with jellyseerr. I enable the perms on my account and jellyseerr shows a section for watchlist. Issue is nothing shows up if I mark it in plex.  Also in logs get a weird can read error. My friend has overseerr and it worked fine so not sure what the deal is. I have tried user perms and global to allow watchlist. I am loggging on with plex user if that matters to overseerr. 

Error:
Plex.tv metadata api. 
errorMessage": "Cannot read properties of undefined (reading 'find')"

**Comment by fallenbagel** [comment]
Please hop onto discord for further troubleshooting as that's where we provide support.

And it looks like this might be a db problem where your db might be messed up

**Reply by faulkkev** [reply]
Thanks I will give it a shot here in a day or two.
